src/scripts/helpers/backbone/views/base.coffee

```diff
@@ -77,11 +77,12 @@ define (require) ->
         historyPage = linksHelper.getCurrentPathComponents().page
 
       if @pageTitle
-        document.title = @pageTitle + settings.titleSuffix
+        pageTitle = @pageTitle.replace(/<\/?[^>]+(>|$)/g, '') # Strip tags for baked HTML titles
+        document.title = pageTitle + settings.titleSuffix
```

Comment:
Will this (setting `document.title`) fix the  title-of-webpage issue? <head><title> equiv?